canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -413,13 +413,13 @@ $palette-search-container-height: 41px;
 			line-height: 46px;  /* Must be same as containing div height */
 			color: $text-secondary;
 			margin-left: 0;
-			margin-right: 10px;
+			margin-right: $spacing-04;
 		}
 	}
 }
 
 .palette-dialog-topbar {
-	height: 41px;
+	height: $spacing-08;
```

Comment:
Even though this is not widely used, we should have it look like it was. Revert since there is no matching token
<img width="900" alt="image" src="https://github.com/user-attachments/assets/ca64b45a-0f71-48f1-bc54-b3051bbf1f8d">
vs master:
<img width="565" alt="image" src="https://github.com/user-attachments/assets/6975d013-afd4-4f22-85c0-ce740e8a523a">